src/App.js

```diff
@@ -1,14 +1,16 @@
 import React from "react";
 import { Container } from "react-bootstrap";
 import "./App.css";
-import Footer from "./components/Footer/Footer";
+//import Footer from "./components/Footer/Footer";
 import "bootstrap/dist/css/bootstrap.min.css";
+import UserProfile from "./components/UserProfile/UserProfile";
 
 function App() {
   return (
     <>
       <Container>
-        <Footer />
+        {/*<Footer />*/}
+        <UserProfile />
```

Comment:
UserProfile **container** should be linked with a router, so it's not rendered every time App is rendered

Reply:
UserProfile will be rendered through user icon on Navbar, right?